test/fixtures/server/valid.js

```diff
@@ -17,4 +17,11 @@
 		b--;
 	}
 
+	// Rule: no-useless-concat
+	// eslint-disable-next-line no-unused-expressions
+	'ab';
+
+	// Rule: prefer-template
+	// eslint-disable-next-line no-unused-expressions
+	`a${global}`;
```

Comment:
`{var}` or `{ var }`? I've always seen and used `{var}`, but the next file in the this uses spaced `{ var }`.

Reply:
Fixed. Added template-curly-spacing.